crates/bevy_diagnostic/src/log_diagnostics_plugin.rs

```diff
@@ -60,18 +60,18 @@ impl LogDiagnosticsPlugin {
                     diagnostic.name,
                     // Suffix is only used for 's' as in seconds currently,
                     // so we reserve one column for it
-                    format!("{:.6}{:1}", value, diagnostic.suffix),
+                    format_args!("{:.6}{:1}", value, diagnostic.suffix),
```

Comment:
Does this right alignment still work? A lot of types don't support alignment and padding. I don't know if `Arguments` is one of the types that supports it.

Reply:
As it turns out, no; removing the `format_args!` layer works great though.